iib/web/api_v1.py

```diff
@@ -376,8 +376,8 @@ def get_builds() -> flask.Response:
 
     if batch_id is not None:
         query_params['batch'] = batch_id
-        batch_id = Batch.validate_batch(batch_id)
-        query = query.filter_by(batch_id=batch_id)
+        batch_id_checked: int = Batch.validate_batch(batch_id)
+        query = query.filter_by(batch_id=batch_id_checked)
```

Comment:
New variable is needed, because of the variable type change from Optional[str] to int by validate_batch function.